applets/clock/clock.c

```diff
@@ -1022,23 +1022,24 @@ create_cities_store (ClockData *cd)
         GtkTreeIter iter;
         GList *cities = cd->locations;
         GList *list = NULL;
+        GList *l;
```

Comment:
You don't need to use a double-linked list since you only use `next` pointer, instead you can use a [GSList](https://developer.gnome.org/glib/stable/glib-Singly-Linked-Lists.html#GSList).

Reply:
It is just an iterator, so has to match the type of the list it points to.  I'm not sure changing `cd->locations` to be a `GSList`, and it anyways is outside the scope of this MR IMO.

Reply:
OK, #1114